src/persistent_list_map.rs

```diff
@@ -306,6 +332,25 @@ mod tests {
         println!("{}", map3);
         println!("{}", map4);
     }
+
+    #[test]
+    fn get_with_default() {
+        let map = persistent_list_map!(map_entry!("x", "v"));
+        let key = Keyword::intern("k").to_rc_value();
+        let default = Keyword::intern("not-found").to_rc_value();
+        let val: Rc<Value> = map.get_with_default(&key, &default);
```

Comment:
I did not intend to leave these type annotations in.

- [ ] remove type annotations in `get_with_default` and `get_with_default_empty` tests